SearchHistory/migrations/0002_test_data.py

```diff
@@ -1,32 +1,35 @@
-from datetime import timedelta
 from django.db import migrations, transaction
-from django.utils import timezone
 
 
 class Migration(migrations.Migration):
 
     dependencies = [
         ('SearchHistory', '0001_initial'),
+        ('SearchHistory', '0002_alter_searchhistory_client_id_and_more'),
+        ('account', '0002_test_data'),
     ]
 
     def generate_data(apps, schema_editor):
         from SearchHistory.models import SearchHistory
-
-        now = timezone.now()
+        from account.models.professional import Professional
+        from account.models.client import Client
 
         test_data = [
-            ('1', '101', '601',  now - timedelta(days=10)),
-            ('2', '102', '602',  now - timedelta(days=20)),
-            ('3', '103', '603',  now - timedelta(days=12)),
-            ('4', '104', '604',  now - timedelta(days=100)),
+            ('1', Professional.filter_by_professional_id(1)[0],
+             Client.filter_by_client_id(2)[0]),
+            ('2', Professional.filter_by_professional_id(1)[0],
+             Client.filter_by_client_id(2)[0]),
+            ('3', Professional.filter_by_professional_id(1)[0],
+             Client.filter_by_client_id(2)[0]),
+            ('4', Professional.filter_by_professional_id(1)[0],
+             Client.filter_by_client_id(2)[0]),
```

Comment:
No, I can't add integers of professional ID instead because professional_id is from models.ForeignKey(Professional, on_delete=models.CASCADE) type. if I add integers of professional ID instead I get errors.
By the way, it's like that in other apps too.